.github/workflows/linting.yml

```
@@ -70,7 +70,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11  # v4.1.1
       - name: Check DAOS ftest tags.
-        run: \[ ! -x src/tests/ftest/tags.py \] || ./src/tests/ftest/tags.py lint
+        run: \[ ! -x src/tests/ftest/tags.py \] || ./src/tests/ftest/tags.py lint --verbose
```

As an aside, just curious about the use of negation and `||` rather than just using `&&`:

```suggestion
        run: \[ -x src/tests/ftest/tags.py \] && ./src/tests/ftest/tags.py lint --verbose
```

I wonder if there is a subtly there that I am just not seeing.

---

No magic here - but me being bad with bash. But I don't think we actually need that check anymore, since `tags.py` should always exist

---

Or maybe so the return is always 0?

---

Ah, yes.  This is the subtlety that I was not seeing.  I wasn't even considering the return status of the pipeline before the second component.